brianteeman/joomla-cms
# Add a message if the filetype is allowed but you are not allowed to show it in the backend

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/9871

#### Summary of Changes

Adds a message if the filetype is supported but not added to the allowed files

#### Testing Instructions

- apply: https://github.com/joomla/joomla-cms/pull/9871
- try to add a new file with the type `sass` or `scss`
- you get a message that the file was added
- but you can't see it.
- apply this patch
- try to add a new file again
- you got a message that you need to add the file type to the allowed filelds.

@brianteeman please have a look into my text proposal for `COM_TEMPLATES_WARNING_FORMAT_WILL_NOT_BE_VISIBLE`

#### Question

Should we add a message if any other created file in the template folder can't be shown because that extension is not added to the allowed fileds?
